dimagi/commcare-hq
# Migration to grant web apps access to default mobile worker roles

## Technical Summary
Branched off of https://github.com/dimagi/commcare-hq/pull/34544/, adds a migration to run that command.

On our prod environments, we should run the management command before merging this, as it may take time to run - see previous PR for detail. Marking this PR as "don't merge" in the meantime.

## Safety Assurance

### Safety story
See https://github.com/dimagi/commcare-hq/pull/34544/

### Automated test coverage

not really

### QA Plan

Not requesting QA, see safety story in https://github.com/dimagi/commcare-hq/pull/34544/

### Migrations
- [x] The migrations in this code can be safely applied first independently of the code

No code other than the migration itself.

### Rollback instructions


Reverting this PR has no impact, as it won't "reverse" the script, and it would not be safe to do so.

### Labels & Review
